Kitodo/src/main/java/org/kitodo/services/file/FileService.java

```diff
@@ -780,7 +783,7 @@ public URI getTemporalDirectory() {
      * @return the URI to the users directory.
      */
     public URI getUsersDirectory() {
-        return Paths.get(ConfigCore.getParameter("dir_Users", "/usr/local/kitodo/users/")).toUri();
+        return Config.getUri(Parameters.DIR_USERS);
```

Comment:
Why did you remove the default value if this parameter is not defined?

Reply:
Because it does usually not work, or even creates merry havoc on the running system. If we provide a built-in default, it must also be a correct one for Windows.